cmd/ketch/testdata/app-log/3.output

```diff
@@ -0,0 +1,16 @@
+[hello-web-1-random/hello-web-1] 2021-01-13 16:49:00 +0000 UTC hello-web-1 0
+[dashboard-worker-2-random/dashboard-worker-2] 2021-01-13 16:49:01 +0000 UTC dashboard-worker-2 0
```

Comment:
Do you think it would be nice to start log with timestamp?
As a user it would be more convenient to me.

Reply:
we are using the same format as `kubectl logs --prefix --timestamps`